docs/guide/tutorial-2-modification-of-the-environment-20-minutes.md

```diff
@@ -185,9 +168,9 @@ The best Geometry primitive to implement the walls is the Box node.
 Only one [Shape](../reference/shape.md) has to be defined for all the walls.
```

Comment:
166: to add an intermediate[Shape] => to add an intermediate [Shape] (missing space)